Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Private/QTMConnectLiveLinkSource.cpp

```diff
@@ -36,13 +36,37 @@ QTMConnectLiveLinkSettings QTMConnectLiveLinkSettings::FromString(const FString&
         if (autoDiscover == "true")
             settings.AutoDiscover = true;
     }
+    settings.Stream3d = true;
+    FString stream3d;
+    if (!FParse::Value(*settingsString, TEXT("Stream3d="), stream3d))
+    {
+        if (stream3d == "false")
+            settings.Stream3d = false;
+    }
+    settings.Stream6d = true;
+    FString stream6d;
+    if (!FParse::Value(*settingsString, TEXT("Stream6d="), stream6d))
+    {
+        if (stream6d == "false")
+            settings.Stream6d = false;
+    }
+    settings.StreamSkeleton = true;
+    FString streamSkeleton;
+    if (!FParse::Value(*settingsString, TEXT("StreamSkeleton="), streamSkeleton))
+    {
+        if (streamSkeleton == "false")
+            settings.StreamSkeleton = false;
+    }
```

Comment:
Default all but `StreamSkeleton` to `false`.

Reply:
Rather stream3d to false... so we don't default stream markers.